test/test.c

```diff
@@ -24242,6 +24242,38 @@ test_JetStreamMgtConsumers(void)
         cfg.FilterSubject = "bar";
     }
 
+#define TIME_20350101 (2051251200L * 1000000000L)
+#define TIME_20350101_STR "\"2035-01-01T08:00:00Z\""
+
+    if (serverVersionAtLeast(2, 11, 0))
+    {
+        test("Add consumer (non durable, paused): ");
+        cfg.PauseUntil = TIME_20350101;
+        s = js_AddConsumer(&ci, js, "MY_STREAM", &cfg, NULL, &jerr);
+        testCond((s == NATS_ERR) && (jerr == JSStreamNotFoundErr) && (ci == NULL));
+        nats_clearLastError();
+
+        test("Verify config: ");
```

Comment:
This is a bit confusing. You try to add a paused consumer above, but the check is that the operation should fail. And here you are checking for the config? Is it that you just want to check the marshaling and see that pause_until is properly set?

Reply:
Cut&pasting prior art FWIW, yes, that appears the intent. I'll move this to after the stream is created, and check the info, as you suggested elsewhere.

Reply:
Moved, create for real now